benchmarks/comparison.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+import shutil
+import subprocess
+import tempfile
+
+from django.conf import settings
+
+compare_script = os.path.join(os.path.dirname(__file__), '../ext/art-testing/compare.py')
+
+def render_comparison(results_before, results_after):
+    tmpdir = export_data(results_before, results_after)
+    # FIXME: sandbox this!
+    output = subprocess.check_output([
+        compare_script,
+        os.path.join(tmpdir, "before.json"),
+        os.path.join(tmpdir, "after.json"),
+    ])
+    shutil.rmtree(tmpdir)
+    return output
+
+
+def export_data(results_before, results_after):
+    tmpdir = tempfile.mkdtemp()
+
+    export_result(results_before, os.path.join(tmpdir, 'before.json'))
+    export_result(results_after, os.path.join(tmpdir, 'after.json'))
+
+    return tmpdir
+
+
+def export_result(results, filename):
```

Comment:
OK, let's leave i like it is for now. If guys don't like it we'll change that. I guess it's connected to our discussion on how to filter results with respect to metadata. If that can be done, the results would be identical as obtained from original JSON results.